pkg/tanka/static.go

```diff
@@ -0,0 +1,62 @@
+package tanka
+
+import (
+	"encoding/json"
+	"log"
+	"path/filepath"
+
+	"github.com/grafana/tanka/pkg/jsonnet/jpath"
+	"github.com/grafana/tanka/pkg/spec"
+	"github.com/grafana/tanka/pkg/spec/v1alpha1"
+)
+
+// StaticLoader loads an environment from a static file called `spec.json`.
+// Jsonnet is evaluated as normal
+type StaticLoader struct{}
+
+func (s StaticLoader) Load(path string, opts JsonnetOpts) (*v1alpha1.Environment, error) {
+	root, base, err := jpath.Dirs(path)
+	if err != nil {
+		return nil, err
+	}
+
+	config, err := parseStaticSpec(root, base)
+	if err != nil {
+		return nil, err
+	}
+
+	data, err := EvalJsonnet(path, opts)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := json.Unmarshal([]byte(data), &config.Data); err != nil {
+		return nil, err
+	}
+
+	return config, nil
+}
+
+// parseSpec parses the `spec.json` of the environment and returns a
```

Comment:
Comment does not match function signature:

```suggestion
// parseStaticSpec parses the `spec.json` of the environment and returns a
```